pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -465,7 +465,6 @@ stdenv.mkDerivation (finalAttrs: {
     (enableFeature withDrm "libdrm")
     (enableFeature withIconv "iconv")
     (enableFeature withJack "libjack")
-    (enableFeature withMfx "libmfx")
```

Comment:
Why?

Reply:
Upon reflection, I'll add this back in because I don't know for certain that vpl handles everything mfx does for older gpus.

mfx and vpl do conflict at configure time, so I will add a nand assertion to make sure they aren't both enabled